Auto Auth Impl

dgunda1 opened the pull request:
Implemented AutoAuthorization Filter for i2b2 integration service

dgunda1 committed:
Auto Auth Impl

Implemented AutoAuthorization Filter for i2b2 integration service

arpost commented:
Can the criteria parser (free marker) code be moved into eurekaclinical-common? It seems like it should not need to be repeated in every service.

dgunda1 committed:
Removed free marker

criteria parser (free marker) code is moved to eureka common
updated the pakage name in ServletModule files
